### ISSUE TITLE
coverageCommand should not be required setting

### ISSUE THREAD

**Issue by insspb** [paambaati/codeclimate-action#182]
**Describe the bug**

When repository uses several coverage sites uploads are made after test runs complete. So no need to rerun coverage.

**Version of `codeclimate-action` you're using**
`v2.5.7`

**Example links**
1. https://github.com/insspb/ikt/pull/29/checks?check_run_id=700558166

**Expected behavior**
In example at the link `echo` command used as placeholder. This should be avoided.

**Comment by CaffeinatedAndroid** [comment]
Yup - an issue for me too.

I perform the following step which runs a command using an emulator:
```yml
    - name: Android tests and coverage
        uses: reactivecircus/android-emulator-runner@v2
        with:
          api-level: 29
          script: ./gradlew createDebugCoverageReport
```

If I were to use `codeclimate-action`, `coverageCommand` would essentially repeat the previous coverage step.

**Comment by paambaati** [comment]
Can you folks help me understand this a little better? Are you saying you've already generated coverage output in a previous step and only need this action to simply upload the coverage output?

**Comment by CaffeinatedAndroid** [comment]
@paambaati Yes.

The command I use to generate the coverage report is required by another GitHub Action. So I use your action just to upload to Code Climate.

For example:
```yml
      - name: Android tests (unit and instrumented)
        uses: reactivecircus/android-emulator-runner@v2
        with:
          api-level: 29
          # Runs: Build, unit tests, instrumented tests, then generates and combines coverage reports
          script: ./gradlew jacocoCombinedTestReports
      - name: Upload coverage to Code-Climate
        uses: paambaati/codeclimate-action@v2.6.0
        env:
          CC_TEST_REPORTER_ID: ${{ secrets.CC_TEST_REPORTER_ID }}
          JACOCO_SOURCE_PATH: "${{github.workspace}}/app/src/main/java"
        with:
          # coverageCommand is required, but as coverage is calculated in previous step, skip by using 'echo'
          coverageCommand: echo "Coverage already calculated - skipping calculating coverage"
          coverageLocations: ${{github.workspace}}/app/build/reports/coverage/jacocoCombinedTestReports/jacocoCombinedTestReports.xml:jacoco
```

**Comment by paambaati** [comment]
This is now available in [`v2.7.1`](https://github.com/paambaati/codeclimate-action/releases/tag/v2.7.1)!
